eq-author-api/utils/datastoreFileSystem.js

```diff
@@ -1,5 +1,6 @@
-const { omit, reject } = require("lodash");
+const { omit, reject, merge } = require("lodash");
```

Comment:
@samiwel are we going to keep this around or should we just drop this filesystem support. It seems an unnecessary burden to maintain.

Reply:
Now that we have some kind of GUI for local dynamodb and an import script to seed the tables, maintaining a filesystem datastore is not as important.